web/docs/deploying.md

```diff
@@ -30,9 +30,16 @@ Below we will explain the required env vars and also provide detailed instructio
 ### Env vars
 
 Server uses following environment variables, so you need to ensure they are set on your hosting provider:
-- `PORT` -> number of port at which it will listen for requests (e.g. `3001`).
-- `DATABASE_URL` -> url to the Postgres database that it should use (e.g. `postgresql://mydbuser:mypass@localhost:5432/nameofmydb`)
-- `JWT_SECRET` -> you need this if you are using Wasp's `auth` feature. Set it to a random string (password), at least 32 characters long.
+- `PORT: int` -> number of port at which it will listen for requests (e.g. `3001`).
+- `DATABASE_URL: string` -> url to the Postgres database that it should use (e.g. `postgresql://mydbuser:mypass@localhost:5432/nameofmydb`).
+- `REACT_APP_URL: string` -> the URL of where the frontend app is running (e.g. `https://<app-name>.netlify.app`), which is used for CORS protection.
```

Comment:
So far in our deployment docs, we tell people to first deploy backend, then frontend. But now, when deploying backend, we actually don't know REACT_APP_URL, because we first need to deploy frontend for it. However the some goes in other direction! Unless we of course know upfront at which URLs they will be deployed, which is much more normal situation.
What I am trying to say is that there is a bit of circularity here, specifically when deploying to heroku and netlify without proper domain and you are not sure which URLs will be used. Should we mention that somehow, or adjust docs for that, warn them, something? Or let them figure it out?

Reply:
Yeah, this is now a circular dependency. I can try to call it out more in the docs (saying something like "If you don't know this offhand now, no problem, we can set it later."), but I am unsure if/how we can get around it, beyond not pointing it out here and then adding a step once the frontend is deployed (which I do kinda mention already).

I am not sure how best to address this to be honest though. 🤔

Reply:
Tried to clarify it a bit more 👍🏻

Reply:
Great!